common/djangoapps/util/testing.py

```diff
@@ -55,3 +57,36 @@ def setUp(self, *args, **kwargs):
 
         self._reset_urls(urlconf_modules)
         self.addCleanup(lambda: self._reset_urls(urlconf_modules))
+
+
+class EventTestMixin(object):
```

Comment:
Is there a way to share the logic with ``EventsTestMixin``? They seem so similar it seems a shame to duplicate them, and their two names are so similar it will be hard to know which one to use when.